source/_changelogs/3.3.0.md

```diff
@@ -0,0 +1,50 @@
+- Your project's current proxy settings now display in the {% url "Settings of the Test Runner" proxy-configuration#View-proxy-settings-in-Cypress %}. Addresses {% issue 3856 %}.
+- We now offer a separate 32bit and 64bit build for Windows. Addresses {% issue 716 %}, {% issue 787 %}, {% issue 2095 %}, {% issue 706 %}.
+- {% url "`.type()`" type %} now supports the special character sequences `{insert}`, `{pageup}`, and `{pagedown}`. Addresses {% issue 3845 %}.
+- We now indicate which XHRs match which alias in the Command Log on {% url "`.wait()`" wait %} when they match an alias multiple times. Addresses {% issue 477 %}.
+- The console log when clicking on the {% url "`cy.visit()`" visit %} command now displays its resolved `options` arguments. Addresses {% issue 3721 %}.
+- There's a new 'canceled' XHR status shown on the Command Log to indicate the difference between calling `xhr.abort()` and the browser canceling the request due to page navigation. Addresses {% issue 3973 %}.
+
+**Bugfixes:**
+
+- Fixed a regression introduced in {% url "3.2.0" changelog#3-2-0 %} that caused an error to throw when visiting domains with certain public suffixes. Fixes {% issue 3717 %}.
+- Fixed a regression introduced in {% url "3.2.0" changelog#3-2-0 %} that caused {% url "`.its()`" its %} to no longer retry when it yields `undefined`. {% issue 3837 %}
+- Fixed a regression introduced in {% url "3.2.0" changelog#3-2-0 %} that caused {% url "`cy.fixture()`" fixture %} to throw a `EISDIR` error if a directory had the same name as a file within that same directory. Fixes {% issue 3739 %}.
+- We changed Cypress's implementation to no longer rely on internal XHR events when the browser navigates. This fixes a a bug in Chrome 71+ that could cause XHRs to pend forever. Fixes {% issue 3973 %}.
+- We greatly improved the speed of proxied requests through our network layer. Fixes {% issue 3192 %} and {% issue 1469 %}.
+- Cypress now properly handles typing JavaScript object prototype methods (like `hasOwnProperty`) into {% url `.type()` type %}. Fixes {% issue 3423 %}.
+- We now set the `form` option to `true` if the `Content-Type` header equals `application/x-www-form-urlencoded` and `json` is not already set by a user. This fixes a situation where Cypress would try to parse the `body` as a string, resulting in a confusing error message. Fixes {% issue 2923 %}.
+- Fixed display of tooltips when selecting elements from the Selector Playground so that they display at the correct coordinates when scrolling. Fixes {% issue 1351 %}.
+- The lists of spec files now display at the correct indentation level when they are nested more than 5 directories deep. Fixes {% issue 3652 %}.
+- Cypress now properly parses environment variables with empty values. Fixes {% issue 3742 %}.
+- Assertions running outside of a test are not properly handled in Cypress instead of throwing internal errors. Fixes {% issue 3926 %}.
```

Comment:
@jennifer-shehane Quick review of this entry: 

> Assertions running outside of a test are _not_ properly handled

Perhaps "now" instead of "not"? Just checking in.

Reply:
🤣🤣🤣🤣 yeah, I think people would like "now" better!